.github/workflows/tests.yml

```diff
@@ -1,7 +1,9 @@
 name: Run all tests
 on:
-  - push
-  - pull_request
+  pull_request:
+      types: [review_requested]
+  push:
+
```

Comment:
Will this still run when a PR is submitted from a fork? I'm guessing that's what the `review_requested` thing tries to do, but what happens when a PR is on a fork, review is requested, then a commit is added on that fork, will the tests run again?

In any case, I like the idea of removing the duplicate run, it's redundant the way we run both on the push and the PR, so I'm glad you're looking into a way to optimize that redundancy out.

Reply:
> Will this still run when a PR is submitted from a fork? I'm guessing that's what the review_requested thing tries to do, but what happens when a PR is on a fork, review is requested, then a commit is added on that fork, will the tests run again?

I am not totally sure. I believe it would though since it's triggered on each push as well. So this way a PR that gets made without any pushes won't get triggered until we request review, but subsequent commits should still trigger the tests

Reply:
I think a push on a fork is not a push on the parent repo, so I don't know if it would get triggered.

I guess the thing to do would be to test it in a dummy fork.